helpers/stylesheet.js

```diff
@@ -4,13 +4,11 @@ const _ = require('lodash');
 const buildCDNHelper = require('./lib/cdnify');
 
 const factory = globals => {
-    return function(assetPath) {
+    return function(assetPath, options) {
```

Comment:
This doesn't allow for html attributes that don't have values (i.e. not passed in via options.hash)